screens/EditBaniOrder.js

```diff
@@ -147,8 +190,10 @@ class Row extends React.Component {
     };
   }
 
+  // eslint-disable-next-line camelcase
   UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Veerji please remove this UNSAFE from here. This is not recommended by official React documentation. 

> Note:
> This method is considered legacy and you should [avoid it](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html) in new code:
> [UNSAFE_componentWillMount()](https://reactjs.org/docs/react-component.html#unsafe_componentwillmount)

Reply:
fixed